features/create_child_topic.feature

```diff
@@ -0,0 +1,20 @@
+#language: fr
+
+Fonctionnalité: Créer des enfants
+
+Contexte:
+Soit l`utilisateur dans la page de modification du point de vue "Histoire de l'art"
+
+Scénario: Créer l'enfant d'un point de vue
+Quand l`utilisateur crée la catégorie "Histoire de l\'art fille" sous le point de vue "Histoire de l\'art"
+Alors la catégorie "Histoire de l\'art fille" apparait dans l`arborescence directement sous "Histoire de l\'art"
+
+Scénario: Créer une fille d'une catégorie
+Quand l`utilisateur crée la catégorie "Histoire de l\'art petite fille" sous "Histoire de l\'art fille"
+Alors la catégorie "Histoire de l\'art petite fille" apparait dans l`arborescence directement sous "Histoire de l\'art fille"
+
+Scénario: Faire de ménage
```

Comment:
What do you really mean by this? Is that a feature of Porphyry?

Reply:
This is to sweep the samples created during the test.